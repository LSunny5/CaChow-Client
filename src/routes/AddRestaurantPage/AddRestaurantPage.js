import React, { Component } from 'react';
import RestaurantForm from '../../components/RestaurantForm/RestaurantForm';
import '../../components/RestaurantForm/RestaurantForm.css';

export default class AddRestaurantPage extends Component {
    render() {
        return (
            <section className='RFormContent'>
                <h2 className="formTitle">Add a Restaurant</h2>
                <RestaurantForm />
            </section >
        )
    };
};