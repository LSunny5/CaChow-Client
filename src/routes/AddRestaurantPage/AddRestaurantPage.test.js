import React from 'react';
import ReactDOM from 'react-dom';
import AddRestaurantPage from './AddRestaurantPage';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { BrowserRouter } from 'react-router-dom';

describe(`AddRestaurantPage route`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <AddRestaurantPage />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the AddRestaurantPage form', () => {
        const wrapper = shallow(<AddRestaurantPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
});