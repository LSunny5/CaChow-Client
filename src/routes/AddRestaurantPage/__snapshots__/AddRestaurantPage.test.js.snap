// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddRestaurantPage route renders the AddRestaurantPage form 1`] = `
<section
  className="RFormContent"
>
  <h2
    className="formTitle"
  >
    Add a Restaurant
  </h2>
  <RestaurantForm />
</section>
`;
